### PR TITLE
chore(*): remove unused modules

### PR DESCRIPTION
**Pull request by djKooks**
There were some warnings during build-testing newest version of project. Please look on.
Thanks.

### CODE REVIEW

File: src/urlencoded.rs [hunk 1]
```diff
@@ -2,7 +2,7 @@ use groupable::Groupable;
 use hyper::uri::RequestUri;
 use hyper::uri::RequestUri::{Star, AbsoluteUri, AbsolutePath, Authority};
 use std::collections::HashMap;
-use url::{form_urlencoded, Url};
+use url::{form_urlencoded};
```
Comment (SimonPersson) [inline src/urlencoded.rs]: No need for curly braces with only one import.

**Comment by SimonPersson** [comment]
Thanks! Could you fix the import I commented on and I'll merge it?

**Comment by djKooks** [comment]
@SimonPersson Thanks~I appreciate.

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were style issues with your Pull Request:

 * Commit: afa14703ddf107ddbe6dab3d311a9119b3a8c695
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/nickel-org/nickel.rs/blob/master/contributing.md

---
This message was auto-generated by https://gitcop.com

**Pull request closed without merging**